satpy/readers/seviri_base.py

```diff
@@ -332,10 +332,15 @@ def images_used(self):
 mpef_product_header = MpefProductHeader().get()
 
 
-class SEVIRICalibrationHandler(object):
-    """Calibration handler for SEVIRI HRIT- and native-formats."""
+class SEVIRICalibrationAlgorithm:
+    """SEVIRI calibration algorithms."""
 
-    def _convert_to_radiance(self, data, gain, offset):
+    def __init__(self, platform_id, scan_time):
+        """Initialize the calibration algorithm."""
+        self.platform_id = platform_id
+        self.scan_time = scan_time
```

Comment:
I'd rather move these arguments to the respective calibration methods, but I didn't want to go over board with the refactoring